test/test_static_vector_kokkos.cpp

```diff
@@ -0,0 +1,113 @@
+#include "ports-of-call/portability.hpp"
+#include "ports-of-call/static_vector.hpp"
+
+#ifndef CATCH_CONFIG_FAST_COMPILE
```

Comment:
I know it's only included if you built with kokkos, but IMO it would still be good to wrap this in `ifdef`s just in case.

Reply:
I'm not sure what `ifdef`(s) you're suggesting, sorry

Reply:
I assume @Yurlungur is referring to the `PORTABILITY_STRATEGY_[KOKKOS|NONE|etc.]` macros.

Reply:
Oh, do you mean to just wrap the whole file in `#ifdef PORTABILITY_STRATEGY_KOKKOS`?

Reply:
yeah that's right